L117-cpp-replacing-gpr-logging-with-abseil-logging.md

```diff
@@ -0,0 +1,94 @@
+L117-Replacing gpr logging with Abseil logging
+----
+
+* Author(s): tjagtap@google.com
+* Approver: roth@google.com , ctiller@google.com
+* Status: In Review 
+* Implemented in: gRPC C++
+* Last updated: April 29, 2024.
+* Discussion at: (filled after thread exists)
+
+## Abstract
+
+Replacing gpr logging and asserts with Abseil logging and asserts
+
+## Background & Rationale
+
+gRPC is currently maintaining its own version of the logging and assert APIs. The Abseil library released their logs and asserts too. We want to leverage the Abseil library and avoid maintaining our own logging code. Also, the current implementation of gpr_log uses format strings which are not type safe and have exploit potential. Moving to absl logging will eliminate this security risk.
+
+## Related Proposals:
+
+None.
+
+## Proposal
+
+We are proposing to remove all instances of gpr logging and asserts and replace them with their absl equivalents.
+
+## Implementation
```

Comment:
Please move this line down to the bottom.  All the text below should be in the "Proposal" section, not the "Implementation" section.  The "Implementation" section should just say that you are making the changes, and you can point to an example PR or two.